docker-compose.override.yml

```diff
@@ -26,26 +26,33 @@ services:
     ports:
       - 8983:8983
 
-  app:
-    volumes:
-      - file_uploads:${UPLOADS_PATH}
-      - derivatives:${DERIVATIVES_PATH}
-      - cache:${CACHE_PATH}
-      - ${HOST_APP_DATA_PATH:-/srv/ngdr/data/}:/data/data
-
   web:
     ports:
       - 3000:3000
     depends_on:
       - cas
     environment:
       - RAILS_FORCE_SSL=false
+    volumes:
```

Comment:
we found that the volume block override wasn't being seen on web and workers under Mac. It's not clear if this is a Mac issue, a docker-compose issue or a yaml parser issue. But basically the yaml <<*app only applies in the file it is declared in (docker-compose.yml) in, it happens first and then gets overridden, so overriding app in this file doesn't actually do anything for web or workers